slime/backends/fsdp_utils/arguments.py

```diff
@@ -26,6 +26,10 @@ class FSDPArgs:
     fsdp_sync_module_states: bool = True
     fsdp_forward_prefetch: bool = True
     fsdp_backward_prefetch: bool = True
+    attn_implementation: str = "flash_attention_2"
```

Comment:
let's just hardcode to `flash_attention_2` since this is the only attn impl that supports data packing.